src/lava_callback.py

```diff
@@ -56,14 +59,34 @@ def hello():
 
 @app.post('/node/<node_id>')
 def callback(node_id):
+    settings = kernelci.settings.Settings(SETTINGS_FILE)
```

Comment:
Settings are now read both here and https://github.com/kernelci/kernelci-pipeline/pull/381/files#diff-b5a78ae1c76918daeeabe1b33399cbda922d09cdf847815dd8c4e0b00e91c0daR19

I'd add at least a TODO entry to align `SETTINGS_FILE` usage throughout this service.

Reply:
Yes, we have "old" way of dealing with settings in pipeline services,  and already @JenySadadia  highlighted that in our development plan.

Reply:
I removed kernelci.settings module for now

Reply:
Do I understand this correctly that temporary removing `kernelci.settings` module and dropping `Settings`/`Secrets` is a transitional step while models are moved around API/Core repositories?

If so, this thread can be marked as resolved but I think it would be good to bring `Secrets` back here once the transition is completed.

Reply:
I follow the suggestions of initial author, @gctucker , on Slack
```
I didn't implement the callback service using kernelci.settings because it wasn't a good idea, it's better to just load the TOML settings directly in this case.  It's not a command-line tool or a service but a Flask application
```
Thats actually good point, as it is web app, it is better to keep it lightweight as possible.
Also using global SETTINGS is better, as it wont re-read and parse file on each request.